docs/operations.rst

```diff
@@ -0,0 +1,10 @@
+Camus Operations
```

Comment:
It's not supposed to display. This is a label so that other parts of the documentation can link to it here. It shouldn't affect the way this page displays at all.